package.json

```diff
@@ -8,9 +8,12 @@
   ],
   "repository": "nodeca/unhomoglyph",
   "license": "MIT",
+  "types": "index.d.ts",
+  "main": "index.js",
```

Comment:
I don't like idea to add `main`. At least, in this PR, because it's not related to TS.

Also not sure "types" field needed.

Reply:
it gets a bit angry if there's `types` but no `main`, but the docs do say that neither is needed if they live at the root named `index.{js|d.ts}` - will remove